charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,289 @@
+        insecure_skip_verify: true
+        metrics: 
+          k8s.container.cpu_limit_utilization:
+            enabled: true 
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: cadvisor
+              bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+              kubernetes_sd_configs:
+                - role: node
+              relabel_configs:
+                - replacement: kubernetes.default.svc.cluster.local:443
+                  target_label: __address__
+                - regex: (.+)
+                  replacement: /api/v1/nodes/$${1}/proxy/metrics/cadvisor
+                  source_labels:
+                    - __meta_kubernetes_node_name
+                  target_label: __metrics_path__
+              scheme: https
+              tls_config:
+                ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                insecure_skip_verify: false
+                server_name: kubernetes
+            - job_name: kubelet
+              bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+              kubernetes_sd_configs:
+                - role: node
+              relabel_configs:
+                - replacement: kubernetes.default.svc.cluster.local:443
+                  target_label: __address__
+                - regex: (.+)
+                  replacement: /api/v1/nodes/$${1}/proxy/metrics
+                  source_labels:
+                    - __meta_kubernetes_node_name
+                  target_label: __metrics_path__
+              scheme: https
+              tls_config:
+                ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                insecure_skip_verify: false
+                server_name: kubernetes
+            # node_exporter job not needed, but leaving configuration in case users want to enable it
+            # - job_name: node_exporter
+            #   kubernetes_sd_configs:
+            #     - role: pod
+            #   relabel_configs:
+            #     - action: keep
+            #       regex: prometheus-node-exporter.*
+            #       source_labels:
+            #         - __meta_kubernetes_pod_label_app_kubernetes_io_name
+            #     - action: replace
+            #       source_labels:
+            #         - __meta_kubernetes_pod_node_name
+            #       target_label: instance
+            #     - action: replace
+            #       source_labels:
+            #         - __meta_kubernetes_namespace
+            #       target_label: namespace
+      filelog:
+        include:
+          - /var/log/pods/*/*/*.log
+          - /var/log/*/*
+          - /var/log/*
+          - /var/log/alternatives.log
+          - /var/log/cloud-init.log
+          - /var/log/auth.log
+          - /var/log/dpkg.log
+          - /var/log/syslog
+          - /var/log/messages
+          - /var/log/secure
+          - /var/log/yum.log
+          - /var/log/alternatives.log
```

Comment:
These are not mounted in the pod, right?

Reply:
no, none of these are mounted in the pod. these paths point to file systems located on the node

Reply:
If there is not `hostPath` mounting these paths from the host to the pod, the collector will not be able to get logs from the host. Am I right?

Reply:
This is still unsolved and I think this is quite important.